app/views/guidance/manage_placements.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= render PageTitle::View.new(text: "Manage your trainee placement data") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: "How to use this service",
+    href: guidance_path,
+    ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">Manage your trainee placement data</h1>
+
+    <p class="govuk-body-m">You are now required to have school placement data in Register.</p>
+
+    <p class="govuk-body-m">This new requirement will be tied to trainees who studied in the <%= @previous_academic_cycle.label %> academic year for performance profiles sign off. </p>
+
+    <p class="govuk-body-m">The performance profile sign off period starts in <%= @performance_profile_sign_off_start_period %> with a deadline of <%= @performance_profile_sign_off_full_deadline %>.</p>
+
+    <h2 class="govuk-heading-m">What type of school placement data is required</h2>
+
+    <p class="govuk-body-m">You only need to provide location information about placements – the URN (if known) or the school name and postcode.</p>
+
+    <p class="govuk-body-m">^ You do not need to provide duration of time spent at each placement</p>
```

Comment:
is `^` doing something worthwhile here?

Reply:
not sure, I just copied all of this directly 🙈

Reply:
it's supposed to be an inset text, updated